ROADMAP-TO-1.0.0.md

```diff
@@ -674,16 +675,16 @@ The go-ipfs effort is the Go implementation of IPFS. It is meant to run as a com
 
 The js-ipfs effort is the javascript implementation of IPFS. It is meant to run on both node.js and today's regular web browsers. It is close, but not yet ready. More at https://github.com/ipfs/js-ipfs
 
-- [ ] js-ipfs ipfs node components
+- [x] js-ipfs ipfs node components
   - [x] js-ipfs multi protocols
   - [x] js-ipfs component: repo
   - [x] js-ipfs component: networking (libp2p)
   - [x] js-ipfs component: bitswap
   - [x] js-ipfs component: dag / ipld
-  - [ ] js-ipfs component: pinning
+  - [x] js-ipfs component: pinning
```

Comment:
Awesome latest addition!

Reply:
wow, when `ipfs.pin.add` in js-ipfs will be released?

Reply:
0.30 🌟 https://github.com/ipfs/js-ipfs/issues/1375